erizo/src/erizo/rtp/RtpExtensionProcessor.cpp

```diff
@@ -137,8 +113,39 @@ uint32_t RtpExtensionProcessor::processAbsSendTime(char* buf) {
   return 0;
 }
 
+void RtpExtensionProcessor::translateExtension(void* extBuffer, const RtpExtentionIdTranslator& translator) {
+  auto* data = reinterpret_cast<char*>(extBuffer);
+  uint8_t src_id = data[0] >> 4;
+  uint8_t len = data[0] & (uint8_t) 0x0f;
+  auto dest_id = (uint8_t) translator.translateId(src_id);
+  if (dest_id != RtpExtentionIdTranslator::padding) {
+    if (dest_id != src_id) {
+      ELOG_DEBUG("Changing extension id from %d to %d", (int) src_id, (int) dest_id);
+    }
+    data[0] = (dest_id << 4) | len;
+  } else {
+    // clear extension
```

Comment:
I guess this solution is good enough as a first step, but we should probably remove the bytes in the header to save bandwidth.

Reply:
In my opinion removing at all bytes from extension will not be a good solution because a single extension is very small (2 or 4 bytes more or less). And because to remove at all an extension you should:
1) change extension length field
2) change packet header length
3) shift following extensions (if needed)
3) add padding (if needed)
4) shift payload
Please consider that to shift payload you can't just use a memcpy(), because on overlapped memory region you have to use memmove() which is slower because internally uses a temporal buffer.
It can be done but in this way we save less then 0.5% of packet length
So I would suggest to use the "padding approach" instead.

Reply:
could you at least put a document with the alternative just in case we need to do it in the future please?